zerver/openapi/python_examples.py

```diff
@@ -1298,13 +1299,14 @@ def test_private_message_invalid_recipient(client: Client) -> None:
 
 
 @openapi_test_function("/messages/{message_id}:patch")
-def update_message(client: Client, message_id: int) -> None:
+def update_message(client: Client, message_id: int, prev_content: str) -> None:
     # {code_example|start}
     # Edit a message. Make sure that `message_id` is set to the ID of the
     # message you wish to update.
     request = {
         "message_id": message_id,
         "content": "New content",
+        "prev_content_sha256": utils.sha256_hash(prev_content),
```

Comment:
This isn't good for the documentation -- most clients won't bother with this, or have a `utils.sha256_hash` function.